spec/ad_hoc_command_spec.rb

```diff
@@ -2,6 +2,7 @@
   let(:url)                 { "example.com/api/v1/ad_hoc_command_update/10" }
   let(:api)                 { AnsibleTowerClient::Api.new(instance_double("Faraday::Connection")) }
   let(:collection)          { api.ad_hoc_commands }
+  let(:described_class)     { api.ad_hoc_command_class }
```

Comment:
I don't like this.  described_class has a *very* well defined meeting and I don't like overriding it in specs.  This is guaranteed to cause confusion.